### PR TITLE
Sinatra

### PR DESCRIPTION
**Pull request by wozane**

### CODE REVIEW

**Comment by wozane** [comment]
@esse request for review

File: spec/test_authenticator.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+
+require_relative '../lib/authenticator.rb'
+
+class TestAuthenticator < MiniTest::Test
+  def test_admin_has_correct_params
+    assert_equal(true, Authenticator.authenticate?('foo', 'bar'))
```
Comment (esse) [inline spec/test_authenticator.rb]: it's better to use simply `assert Authenticator.authenticate?('foo', 'bar')`, it's equivalent

File: spec/test_authenticator.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+  def test_admin_with_incorrect_params
+    assert_equal(false, Authenticator.authenticate?('foo', 'poo'))
```
Comment (esse) [inline spec/test_authenticator.rb]: it's better to use simply `assert !Authenticator.authenticate?('foo', 'poo')`, it's equivalent

File: wozane.rb [hunk 3]
```diff
@@ -1,25 +1,73 @@
...
+      session[:admin] = true
+      redirect to('/admin')
+    else
+      flash[:alert] = 'Try again!'
+      redirect to('/login')
+    end
+  end
+
+  get '/admin' do
+    erb :admin
```
Comment (esse) [inline wozane.rb]: I would love to see here authentication check ;)

**Review by esse — CHANGES_REQUESTED** [review]
Final touch

File: spec/test_authenticator.rb (hunk 2 above)
Comment (wozane) [reply spec/test_authenticator.rb]: 👍

File: spec/test_authenticator.rb (hunk 1 above)
Comment (wozane) [reply spec/test_authenticator.rb]: 👍

File: wozane.rb (hunk 3 above)
Comment (wozane) [reply wozane.rb]: @esse please check if this can work

**Review by esse — APPROVED** [review]

**Pull request merged**